map_processing/data_models.py

```diff
@@ -89,6 +139,26 @@ def _validator_for_numpy_array_deserialization(v: Union[str, np.ndarray]) -> np.
 
 
 class Weights(BaseModel):
+    """
+    A representation of weights for all parameters from data
+
+    Attributes:
+        orig_gravity: A numpy ndarray representing initial gravity weights
```

Comment:
Include the length of each weight vector (e.g., "orig_gravity: A 3-element numpy ndarray representing initial gravity weights")